src/Polysemy/State.hs

```diff
@@ -15,6 +15,7 @@ module Polysemy.State
     -- * Interpretations
   , runState
   , evalState
+  , execState
   , runLazyState
```

Comment:
Please also provide an `execLazyState` for symmetry.

Reply:
Will do.